### ISSUE TITLE
Will 2.1.0 regex matcher seems broken 

### ISSUE THREAD

**Issue by wontonst** [skoczen/will#327]
![image](https://user-images.githubusercontent.com/2030556/34002551-b77fcd2a-e0a7-11e7-876d-e0b6602c163b.png)

![image](https://user-images.githubusercontent.com/2030556/34002816-83ac6b60-e0a8-11e7-8e37-abcd507eb102.png)

It's a very amusing bug though, everyone on my team enjoyed this great great feature bug.

```
will==2.1.0
will-natural==0.2.1.1
```

Does not occur on will 1.0.2

![image](https://user-images.githubusercontent.com/2030556/34002872-a6ffe36c-e0a8-11e7-8833-377a1b63670e.png)

**Comment by CodingWithKyle** [comment]
Hey, we ran into this issue as well. For us it was resolved by disabling the "Fuzzy" searches being turned on in config.py - but I think it was on in the past. So maybe this is some unexpected new behavior?

**Comment by skoczen** [comment]
Fuzzy matching is new in 2.x - thanks for the report - sounds like the default level of fuzziness (or a minimum char limit) would help this out!

**Comment by skoczen** [comment]
(Quick paste-on-all-open-issues update:)

**TL;DR**: New Will release EOM. Also, you're awesome.

**Full Update:** This may be TMI for you, and if so, cool, the thing you probably care about is above.  But given the total radio silence on my part - and the fact that I've always tried to create a community here where we got to be actual people instead of avatars and PRs, I wanted to give a bit more context.

As you may be aware from an earlier AFK, I got married in December, which was great, but burned the month from a Will perspective.  I planned to get back on it at the start of January, but instead found myself selling our house, and simultaneously hit with a surprisingly severe bout of depression.  The time since Jan 1 has been sort of a blur.  A dark blur, but a blur more than anything.   I saw the PRs and comments coming in, but didn't have the energy or time to do anything about them.

Things seem to be clearing up on both counts now, and I'm _really_ looking forward to getting these fixes in, catching up on bug reports, and being active in improving Will again. :)

With that, I'm aiming to get all outstanding PRs bundled into a release, and all issues replied to by the end of February.

Most of all, thanks for your contribution to Will, your patience with an imperfect, quite human project maintainer, and for continuing to come back, engage, and spend some of the precious seconds you get on earth making this project better.

I appreciate you.  Thank you.

**Comment by skoczen** [comment]
Note for me for 2.1.1: test out these examples, tweak fuzziness or minimum char matches so they don't happen.

**Comment by skoczen** [comment]
@frypan-handler @wontonst the fix for this is simply to tweak config.py to

```python
FUZZY_MINIMUM_MATCH_CONFIDENCE = 91
```

This will be the default in 2.2.1, but if your file currently is set to 90, you'll need to override.

Side note: this is easily the most enjoyable ~~pug~~ bug I've ever tested and fixed on Will.  Thanks for that. :)

**Comment by skoczen** [comment]
Sadly, this has been fixed, and released in 2.1.1. :)
